packages/evm/test/Enclave.spec.ts

```diff
@@ -541,24 +594,25 @@ describe("Enclave", function () {
         .to.be.revertedWithCustomError(enclave, "InvalidEncryptionScheme")
         .withArgs(encryptionSchemeId);
     });
-    it("reverts if given E3 Program does not return input validator address", async function () {
-      const { enclave, mocks, owner, request } = await loadFixture(setup);
-      await mocks.e3Program
-        .connect(owner)
-        .setInputValidator(ethers.ZeroAddress);
-      await expect(
-        enclave.request(
-          request.filter,
-          request.threshold,
-          request.startTime,
-          request.duration,
-          request.e3Program,
-          request.e3ProgramParams,
-          request.computeProviderParams,
-          { value: 10 },
-        ),
-      ).to.be.revertedWithCustomError(enclave, "InvalidComputationRequest");
-    });
+    // XXX: work out a different way to test this
```

Comment:
nit: maybe add a //todo comment here?

Reply:
I have resolved this and reinstated the test.